central/complianceoperator/v2/report/manager/manager_impl.go

```diff
@@ -496,13 +514,39 @@ func (m *managerImpl) handleReportScheduled(request *reportRequest, isOnDemand b
 
 func (m *managerImpl) validateScanConfigResults(result *watcher.ScanConfigWatcherResults) error {
 	if result.Error != nil {
-		return result.Error
+		if errors.Is(result.Error, watcher.ErrScanConfigTimeout) {
+			return utils.ErrScanConfigWatcherTimeout
+		}
+		return utils.ErrScanWatchersFailed
```

Comment:
It looks like this error is not logged nor wrapped. Are we able to find the contents of `result.Error` in a different place?